frontend/appflowy_flutter/lib/plugins/document/application/document_bloc.dart

```diff
@@ -40,12 +40,16 @@ part 'document_bloc.freezed.dart';
 
 bool enableDocumentInternalLog = false;
 
+final Map<String, DocumentBloc> documentBlocMap = {};
```

Comment:
If we do it like this, we can avoid mutability outside of this file:
```
final Map<String, DocumentBloc> _documentBlocs = {};

class DocumentBloc extends Bloc<DocumentEvent, DocumentState> {
  static DocumentBloc? for(String documentId) => _documentBlocs[documentId];

  ...,
}
```

Reply:
Thanks, I've added this change. I named the method `findOpen` instead because `for` is a dart keyword.